frontend/src/entry/model/settings/TemplateSettings.tsx

```diff
@@ -0,0 +1,63 @@
+import { LoadingButton } from '@mui/lab'
+import { Checkbox, Divider, FormControlLabel, Stack, Typography } from '@mui/material'
+import { patchModel } from 'actions/model'
+import { useState } from 'react'
+import useNotification from 'src/hooks/useNotification'
+import MessageAlert from 'src/MessageAlert'
+import { EntryInterface } from 'types/types'
+import { getErrorMessage } from 'utils/fetcher'
+
+type TemplateSettingsProps = {
+  model: EntryInterface
+}
+
+export default function TemplateSettings({ model }: TemplateSettingsProps) {
+  const [allowTemplate, setAllowTemplate] = useState(model.settings.allowTemplating)
+  const [loading, setLoading] = useState(false)
+  const [errorMessage, setErrorMessage] = useState('')
+  const sendNotification = useNotification()
+
+  async function handleSave() {
+    setLoading(true)
+    const updatedModelSettings = {
+      settings: {
+        ungovernedAccess: false,
+        allowTemplating: allowTemplate,
```

Comment:
If you rename `allowTemplate` to `allowTemplating` as I've suggested in another comment, this can be changed from `allowTemplating: allowTemplate,` to `allowTemplating`.

Reply:
Unresolved as this is still outstanding